server/routes/config/licenceConditions.js

```diff
@@ -1,4 +1,15 @@
 module.exports = {
+    standardConditions: {
+        nextPathDecision: {
+            fieldToDecideOn: 'additionalConditions',
+            Yes: '/hdc/licenceConditions/additionalConditions/',
+            No: '/hdc/licenceConditions/riskManagement/'
+        }
+    },
+    conditionsReview: {
+        licenceSection: 'additionalConditions',
+        nextPath: '/hdc/licenceConditions/riskManagement/'
+    },
     riskManagement: {
```

Comment:
I'm not convinced about having risk management as part of licence conditions but it seems I'm behind the times re state-of-the-art HDC routing logic magic

Reply:
Probably agree. The original idea was to grop them all together in the 5 journeys. Will be worth revisiting afterwards I think.